src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java

```diff
@@ -17,16 +18,26 @@
     @FindBy(id = "personnelMenu")
     private WebElement personnelMenu;
 
+    @FindBy(id = "ppeMenu")
+    private WebElement ppeMenu;
+
     @FindBy(id = "employeeMenu")
     private WebElement employeeMenu;
 
     @FindBy(id = "roleMenu")
     private WebElement roleMenu;
 
+    @FindBy(id = "ppeClassificationSubMenu")
+    private WebElement ppeClassificationSubMenu;
+
     public void clickOnPersonnelMenu(){
```

Comment:
user clickPersonnelMenu instead of clickOnPersonnelMenu, for all method in this class